src/routes/docs/quick-starts/apple/+page.markdoc

```diff
@@ -56,39 +56,9 @@ Once the SDK is found, select **Up to Next Major Version** as your **Dependency
 
 When dependency resolution is complete, click **Add Package** again to add the SDK package to your target.
 
-In order to allow creating OAuth sessions, the following URL scheme must be added to your **Info.plist** file.
-
-```xml
-<key>CFBundleURLTypes</key>
-<array>
-<dict>
-    <key>CFBundleTypeRole</key>
-    <string>Editor</string>
-    <key>CFBundleURLName</key>
-    <string>io.appwrite</string>
-    <key>CFBundleURLSchemes</key>
-    <array>
-        <string>appwrite-callback-[PROJECT_ID]</string>
-    </array>
-</dict>
-</array>
-```
-
-If you're using UIKit as opposed to SwiftUI, you will also need to add the following to your **SceneDelegate.swift** file.
-
-```swift
-func scene(_ scene: UIScene, openURLContexts URLContexts: Set<UIOpenURLContext>) {
-    guard let url = URLContexts.first?.url,
-        url.absoluteString.contains("appwrite-callback") else {
-        return
-    }
-
-    WebAuthComponent.handleIncomingCookie(from: url)
-}
-```
```

Comment:
Let's keep this here

Reply:
I thought it might be a good idea to remove this since it adds unnecessary steps to get the user up and running with   the `createEmailSession` flow. Wouldn't leaving the code there cause some confusion for newcomers to Appwrite?

Reply:
I agree it's unnecessary if you only want to create an email/password flow, but I think it's important to have it here for users who do want to use OAuth; this information is otherwise a bit harder to find. What do you think @gewenyu99?

Reply:
Bumping - @gewenyu99